hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/WithObjectID.java

```diff
@@ -22,34 +22,24 @@
 /**
  * Mixin class to handle ObjectID and UpdateID.
  */
-public class WithObjectID extends WithMetadata {
+public abstract class WithObjectID extends WithMetadata {
```

Comment:
Any reason for marking these abstract? Is it because don't want to instantiate this.  If that is the case, WithParentObjectId is not marked abstract.

Reply:
Yes, these are intended to be subclassed, not instantiated.  This part of the change is not critical, though.

`WithParentObjectId` is instantiated in `PrefixParser`, so I couldn't make it `abstract` without further change.

Reply:
Got it, Thanks.